code/modules/mob/transform_procs.dm

```diff
@@ -469,6 +470,11 @@
 		else if(O.ai_controller)
 			QDEL_NULL(O.ai_controller)
 
+	if(O.dna?.species && !istype(O.dna.species, /datum/species/monkey))
+		O.set_species(O.dna.species)
+	else
+		O.set_species(/datum/species/human)
```

Comment:
if they don't have dna set species to human? 🤔

Reply:
Yes. Humanize is a fucking cursed proc.

Reply:
This will runtime

Reply:
It doesn't. I tested this part alot.

Reply:
set_species won't do anything if mob has no dna.

Assume it has the dna and don't check for it ~2 lines above.